backends/platform/sdl/sdl.cpp

```diff
@@ -64,6 +64,11 @@
 #include <SDL/SDL_net.h>
 #endif
 
+#if SDL_VERSION_ATLEAST(2, 0, 0)
```

Comment:
This looks weird - the SDL_VERSION_ATLEAST macro should only be known if you already included SDL.h.

also (not sure about the scummvm guidelines) it looks wrong to include SDL2/SDL.h - just include SDL.h and let the include paths be configured correctly (e.g. pkg-config --cflags sdl2 will print /usr/include/SDL2 on my system - and this is also the default that SDL2 configures in their pc files). If you already include SDL.h for version >= 2.0.0, SDL_clipboard.h is also automatically included.